src/lib/Siteaccess/SiteaccessResolverInterface.php

```diff
@@ -28,7 +28,18 @@ public function getSiteaccessesForLocation(
     ): array;
 
     /**
-     * Returns complete list of siteaccesses.
+     * Returns a complete list of Site Access objects.
+     *
+     * @return \eZ\Publish\Core\MVC\Symfony\SiteAccess[]
+     */
+    public function getSiteAccessesListForLocation(
+        Location $location,
+        int $versionNo = null,
```

Comment:
both should be nullable

```suggestion
        ?int $versionNo = null,
```